tcms/telemetry/static/telemetry/css/testing/status-matrix.css

```diff
@@ -0,0 +1,21 @@
+.table-with-horizontal-scroll {
+    display: block;
+    overflow-x: auto;
+    white-space: nowrap;
+}
+
+table > tbody > tr > td.pass {
```

Comment:
Make CodeClimate happy by not introducing any more issues than necessary.

Reply:
Done